docs/Home.md

```diff
@@ -7,3 +7,4 @@ This repository hosts the build, test and release scripts for OpenSearch and Ope
 See each page below for more details:
 - [Building a Distribution from the source](https://github.com/opensearch-project/opensearch-build/wiki/Building-an-OpenSearch-and-OpenSearch-Dashboards-Distribution)
 - [Testing a Distribution](https://github.com/opensearch-project/opensearch-build/wiki/Testing-the-Distribution)
+- [Releasing the Distribution](https://github.com/opensearch-project/opensearch-build/wiki/Testing-the-Distribution)
```

Comment:
Should this be updated to `Releasing-the-Distribution`?

Reply:
These are home page headings https://github.com/opensearch-project/opensearch-build/wiki